pkg/apiserver/registry/networkpolicy/clustergroupmember/rest.go

```diff
@@ -44,15 +46,27 @@ type groupMembershipQuerier interface {
 	GetGroupMembers(name string) (controlplane.GroupMemberSet, []controlplane.IPBlock, error)
 }
 
+type paginationInfo struct {
+	pageNumber int // stores current page number from query, defaults to 0
+	pageLimit  int // stores page limit from query, defaults to 0
```

Comment:
I assume that means all the items will be returned at once? Is there a comment about this somewhere?

Reply:
Modified the comments in `paginateMemberList` function, figured the default value comment should be at initiation of `paginationInfo` instead of here in struct. Thanks.